templates/reloader.yaml

```
@@ -0,0 +1,29 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: reloader
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: reloader
+  annotations:
+    appversion.kubeaddons.mesosphere.io/dex: "v0.0.49"
+    values.chart.helm.kubeaddons.mesosphere.io/reloader: https://raw.githubusercontent.com/stakater/Reloader/ded923b12a17a0f5c9f37f1c4594331d8d22fa70/deployments/kubernetes/chart/reloader/values.yaml
+    # catalog.kubeaddons.mesosphere.io/addon-revision: "0.0.49-1"
```

This has to be present to keep this repository compatible with the Kubeaddons Catalog:
```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "0.0.49-1"
```